src/cargo/ops/mod.rs

```diff
@@ -22,11 +22,14 @@ pub use self::registry::{registry_login, search, http_proxy_exists, http_handle}
 pub use self::registry::{modify_owners, yank, OwnersOptions};
 pub use self::cargo_fetch::{fetch};
 pub use self::cargo_pkgid::pkgid;
-pub use self::resolve::{resolve_pkg, resolve_with_previous};
+pub use self::resolve::{resolve_pkg, resolve_with_previous, source_ids_from_config};
+pub use self::cargo_output_metadata::{
+	output_metadata, OutputTo, OutputFormat, OutputMetadataOptions};
```

Comment:
Stylistically in Cargo imports are at most 80 characters long and a new line means adding a new `pub use self::...` line.